pages/stats.tsx

```diff
@@ -129,7 +128,12 @@ export default () => {
 
   const targetGradeDistribution = React.useMemo(() => {
       const map = new Map<string, number>();
-      soll.data.forEach((item) => {
+      // initialize all grades with count 0 for gyms without targets
+      grades.forEach((grade) => {
+        map.set(grade, 0);
+      });
```

Comment:
Non-imperative way to initialize a Map
```suggestion
      // initialize all grades with count 0 for gyms without targets
      const map = new Map<string, number>(grades.map(grade => [grade, 0]));
```

Reply:
Maybe `as const` is needed, to make TS happy.

```typescript
const map = new Map<string, number>(grades.map(grade => [grade, 0] as const));
```